src/Handler/AddPrefixCommandHandler.php

```diff
@@ -58,43 +58,65 @@ public function __construct()
      *
      * @return int Returns 0 on success and a positive integer on error.
      */
-    public function handle(Args $args, IO $io)
+    public function handle($prefix, array $paths, BasicFormatter $formatter)
```

Comment:
- phpdoc is outdated
- what's prefix type?
- I would add a phpdoc block for `$paths` as well which I guess is a `string[]`